guard/README.md

```diff
@@ -265,6 +265,38 @@ You can write a YAML-formatted unit test file (test.yml) as:
 You can then test your rules file using the `test` command as:
 
 ```bash
-$ cfn-guard test -r rules.guard -t test.yml 
+$ cfn-guard test -r rules.guard -t test.yml
 PASS Expected Rule = assert_all_resources_have_non_empty_tags, Status = SKIP, Got Status = SKIP
 PASS Expected Rule = assert_all_resources_have_non_empty_tags, Status = FAIL, Got Status = FAIL
+```
+
+### Autocompletions
+
+To setup Autocompletions you will need to follow instructions for the specific shell your are running.
+
+Currently guard only supports autocompletions for zsh, bash, and fish shells. If you would like autocompletions for a specific shell feel free to open up a new github issue.
+
+Autocompletions are only something available for version >= 3.0
+
+#### zsh
+
+```sh
+    cfn-guard --completions --shell='zsh' > usr/local/share/zsh/site-functions/_cfn-guard && compinnit
```

Comment:
Is the first slash missing before `usr`? Shouldn't it be `/usr...`?

Reply:
The following seemed to work
```
cfn-guard completions --shell='zsh' > /usr/local/share/zsh/site-functions/_cfn-guard && compinit
```

Reply:
Can we format the descriptions to look like a table? (if only this is straightforward) is there a way for descriptions to not overflow where we are listing commands?

### Current
```
help        -- Print this message or the help of the given subcommand(s)
migrate     -- Migrates 1.0 rules to 2.0 compatible rules.
parse-tree  -- Prints out the parse tree for the rules defined in the file.
rulegen     -- Autogenerate rules from an existing JSON- or YAML- formatted data. (Currently works with only CloudFormation templates)
test        -- Built in unit testing capability to validate a Guard rules file against
unit tests specified in YAML format to determine each individual rule's suc
validate    -- Evaluates rules against the data files to determine success or failure.
You can point rules flag to a rules directory and point data flag to a data
```

### Expected (notice the line below test command)
```
help        -- Print this message or the help of the given subcommand(s)
migrate     -- Migrates 1.0 rules to 2.0 compatible rules.
parse-tree  -- Prints out the parse tree for the rules defined in the file.
rulegen     -- Autogenerate rules from an existing JSON- or YAML- formatted data. (Currently works with only CloudFormation templates)
test        -- Built in unit testing capability to validate a Guard rules file against
               unit tests specified in YAML format to determine each individual rule's suc
validate    -- Evaluates rules against the data files to determine success or failure.
You can point rules flag to a rules directory and point data flag to a data
```